src/main/java/data/workouts/WorkoutList.java

```diff
@@ -41,4 +42,34 @@ public Workout createAndAddWorkout(String userArgument) throws ArrayIndexOutOfBo
     public boolean checkIfRepsValueIsValid(int userRepsInput) {
         return userRepsInput >= 1;
     }
+
+    public void listWorkout(){
+        int index = 0;
+        int totalPrints = workoutsList.size();
+
+        while (totalPrints > 10){
```

Comment:
Minor formatting issue - place space between ) and {.

Also might want to consider using constants instead of the number 10.

Reply:
Good suggestion! Thanks